src/sentry/conf/server.py

```diff
@@ -1341,6 +1342,12 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     "organizations:crons-timeline-listing-page": False,
     # Enable usage of customer domains on the frontend
     "organizations:customer-domains": False,
+    # Enable Discord integration
```

Comment:
Are these from rebasing?

Reply:
Yup

Reply:
Usually when you rebase and deal w/ a merge conflict changes that others made shouldn't show as new changes in your files